### ISSUE TITLE
Missing Capability NET_RAW in manifest?

### ISSUE THREAD

**Issue by mhuttner** [plunder-app/kube-vip#144]
I tried running kube-vip as a static Pod using ARP
 - OS/Distro: Ubunutu20.04
 - Kubernetes Version: v1.19
 - cri-o Version: v1.19
 - Kube-vip Version: 0.2.3

After bootstraping the cluster the ARP Broadcast fails with 
```
time="2020-12-18T13:52:10Z" level=info msg="Broadcasting ARP update for 132.199.249.2 (b0:26:28:f1:e2:a6) via eno3"
time="2020-12-18T13:52:10Z" level=warning msg="failed to get raw socket: operation not permitted"
```

I tracked this down to the `NET_RAW` Capability and after adding it to 
```yaml
 securityContext:
      capabilities:
        add:
        - NET_ADMIN
        - SYS_TIME
        - NET_RAW
```
in the manifest everything seems to work.

Not sure if this should be the default, feel free to close this if this is just needed for my setup.
But maybe this is useful for someone else. 

**`Kube-vip.yaml`:**
```yaml
apiVersion: v1
kind: Pod
metadata:
  creationTimestamp: null
  name: kube-vip
  namespace: kube-system
spec:
  containers:
  - args:
    - start
    env:
    - name: vip_arp
      value: "true"
    - name: vip_interface
      value: eno3
    - name: vip_address
      value: <censored>
    - name: vip_leaderelection
      value: "true"
    - name: vip_leaseduration
      value: "5"
    - name: vip_renewdeadline
      value: "3"
    - name: vip_retryperiod
      value: "1"
    image: plndr/kube-vip:0.1.8
    imagePullPolicy: Always
    name: kube-vip
    resources: {}
    securityContext:
      capabilities:
        add:
        - NET_ADMIN
        - SYS_TIME
    volumeMounts:
    - mountPath: /etc/kubernetes/admin.conf
      name: kubeconfig
    - mountPath: /etc/ssl/certs
      name: ca-certs
      readOnly: true
  hostNetwork: true
  volumes:
  - hostPath:
      path: /etc/kubernetes/admin.conf
    name: kubeconfig
  - hostPath:
      path: /etc/ssl/certs
    name: ca-certs
status: {}
```

**Comment by thebsdbox** [comment]
This is odd, i've never had anyone report this as an issue before.

https://man7.org/linux/man-pages/man7/capabilities.7.html
`NET_ADMIN`, has normally been fine for modifying interfaces.. I wonder if this is specific to the ubuntu 2004 kernel.

**Comment by thebsdbox** [comment]
I appreciate the testing btw!, I'm happy to add this capability to the manifests. The majority of "scary" capabilities are in the `NET_ADMIN` cap anyway.

**Comment by mhuttner** [comment]
Yeah i did not find any other bug reports for this error, and i think my setup is pretty common. 
I am running the 5.4.0-58-generic kernel. 
Thanks for the quick response!

**Comment by thebsdbox** [comment]
Fixed in #148 (along with CI)
